imports/plugins/core/orders/server/no-meteor/util/buildOrderFulfillmentGroupFromInput.js

```diff
@@ -0,0 +1,76 @@
+import Random from "@reactioncommerce/random";
+import buildOrderItem from "./buildOrderItem";
+import updateGroupTotals from "./updateGroupTotals";
+
+/**
+ * @summary Builds an order fulfillment group from fulfillment group input.
+ * @param {Object} context an object containing the per-request state
+ * @param {Object[]} [additionalItems] Additional already-created order items to push into the group
+ *   items array before calculating shipping, tax, surcharges, and totals.
+ * @param {Object} [billingAddress] The primary billing address for the order, if known
+ * @param {String|null} [cartId] ID of the cart from which the order is being placed, if applicable
+ * @param {Object} currencyCode Currency code for all money values
```

Comment:
Should `currencyCode` be a `string` instead of an `object?

<img width="220" alt="GraphiQL" src="https://user-images.githubusercontent.com/4482263/54232831-d76e7880-44c8-11e9-806c-53216eedf9ba.png">